docs/dev/privacy-model-of-secret-contracts.md

```diff
@@ -39,6 +39,29 @@ Secret Contract developers must always consider the trade-off between privacy, u
   - [Differences in output events](#differences-in-output-events)
   - [Differences in output types - success vs. error](#differences-in-output-types---success-vs-error)
 
+# Verified Values During Contract Execution
+
+During execution, some contracts may want to use "external-data" - meaning data that is generated outside of the enclave and sent into the enclave - such as the tx sender address, the funds sent with the tx, block height, etc..
+As these parameters get sent to the enclave, they can theoretically be tampered with, and an attacker might send false data.
+Thus, relying on such data might be risky.
+
+As an example, let's say we are implementing an admin interface for a contract, i.e. functionality that is open only for a predefined address.
+In that case, we want to know that the `env.message.sender` parameter that is given during contract execution is legit, then we want to check that `env.message.sender == predefined_address` and provide admin functionality if that condition is met.
+If the `env.message.sender` parameter can be tampered with - we effectively can't rely on it and cannot implement the admin interface.
+
+## Tx Parameter Verification
+Some parameters are easier to verify, but for others it is less trivial to do so. Exact details about individual parameters are detailed further in this document.
```

Comment:
This section is a very good high-level explanation of the limitations that contract authors need to be aware of, but it's missing the punchline of "this is what we know how to absolutely verify, but that thing we don't"

Reply:
This is the purpose of the following table:
https://github.com/enigmampc/SecretNetwork/blob/2aa31ad892fba390226665b3d29d021b0f6c6143/docs/dev/privacy-model-of-secret-contracts.md#inputs

Reply:
Ahhh, true true, fair enough